CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-Unreleased
-----------
-
+13.3.0
+------
 * Added Payload Verification module [#992](https://github.com/Shopify/shopify_app/pull/992)
+* Added Rubocop to CI
```

Comment:
Since RuboCop doesn't impact users of the app, perhaps leave it out of the changelog?

Reply:
Thanks for clarifying! I've gone ahead and removed it.